en/2-deeds.md

```diff
@@ -24,107 +24,107 @@ There are four types of Deeds, each providing different capabilities in terms of
 
 ## 2.3. Deeds ownership
 
-A Deed gives its owner the right to USE it, RENT it, or SELL it.
+A Deed gives its owner the right to USE, RENT, or SELL it.
 
-- USE – The owner can use the Deed to run a hub for his own organization.
+- USE – The owner can use the Deed to run a hub for his organization.
 - RENT - The owner can also rent the Deed to another organization and earn passive income.
 - SELL - The owner can sell it on a secondary NFT marketplace such as [OpenSea](https://opensea.io).
 
-Being an active holder or renter of a Deed, the organization is entitled to MEED rewards according to the “mintium” model described below (see 2.6).
+Being an active holder or renter of a Deed, the organization is entitled to MEED rewards according to the "Mintium" model described below (see 2.6).
 
 ## 2.4. Community Hubs
 
-In today's competitive market, organizations seek to attract and retain top talent not only through compensation but also by enhancing work engagement. The WoM addresses this challenge with a unique, tokenized reward system embodied in Community Hubs.
+Organizations seek to attract and retain top talent in today's competitive market through compensation and enhancing work engagement. The WoM addresses this challenge with a unique, tokenized reward system embodied in Community Hubs.
 
-Within these Hubs, community leaders can drive members engagement by offering a transparent menu of incentives for work items. Participants earn engagement points for their efforts, which are periodically converted into MEED tokens or a custom equivalent, in line with a predetermined budget.
+Within these Hubs, community leaders can drive members' engagement by offering a transparent menu of incentives for work items. Participants earn engagement points for their efforts. The system then periodically converts these points into reward tokens per a predetermined budget.
 
 Community Hubs offer a suite of tools and features designed to boost and manage contributor engagement:
```

Comment:
Simplified English, but the word"offer" should be replaced with "offers"

Reply:
> Simplified English, but the word"offer" should be replaced with "offers"

No. "Community Hubs offer"  plural